templates/gcp/pro.html

```diff
@@ -0,0 +1,184 @@
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h2>What's in Ubuntu Pro?</h2>
+      <ul class="p-matrix u-clearfix">
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">FIPS & CC-EAL2 certified</h3>
+            <div class="p-matrix__desc">
+              <p>Officially certified components to enable operating environments under compliance regimes like FedRAMP, HIPAA, PCI and ISO.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">CIS and DISA STIG profiles</h3>
+            <div class="p-matrix__desc">
+              <p>For companies looking to leverage industry benchmarks for hardening, Ubuntu Pro makes two leading profiles available.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Open source security</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro adds security coverage for the most important open source applications like Apache Kafka, NGINX, MongoDB, Redis and PostgreSQL.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Predictable and optimised</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro is based on Ubuntu and leverages both the breadth of open source included and the GCP optimisations built in.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Kernel Livepatch</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu kernel updates are regularly issued. With our Livepatch service, kernel patches are delivered immediately, without the need to reboot your VMs.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Priced for the cloud</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro pricing tracks the underlying compute cost, starting at under $0.01 for the smallest B1ls instance type, and ramping down to less than 1% of hourly compute.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">The right Ubuntu for you</h3>
+            <div class="p-matrix__desc">
+              <p>Canonical brings the three most popular Ubuntu Server distributions to choose from - 14.04 LTS, 16.04 LTS, 18.04 LTS and 20.04 LTS.</p>
```

Comment:
This is different from the copy doc.

Reply:
Can you encode the dash with an `&mdash;` please

Reply:
Can you add a `&nbsp;` between the version and LTS please

Reply:
All done